content/src/editor.less

```diff
@@ -805,19 +818,34 @@ body#pencildoc #diigolet-csm {
 }
 
 .directory .item {
+  text-align: center;
   display: inline-block;
   float: left;
   clear: left;
   text-decoration: none;
-  padding: 0 28px 0 12px;
-  min-height: 28px;
   color: #222;
-  cursor: pointer
+  cursor: pointer;
+  div {
+    width: 200px;
```

Comment:
If we are using a 100x100 square for displaying thumbnails, do we still want to save the thumbnail in the size of 128x128?